DigitalLearningSolutions.Data/DataServices/CourseDataService.cs

```diff
@@ -533,7 +541,8 @@ FROM CentreApplications
                             )
                             THEN CAST(1 AS BIT)
                             ELSE CAST(0 AS BIT)
-                        END AS CentreHasApplication
+                        END AS CentreHasApplication,
+                        a.DefaultContentTypeID AS DefaultContentTypeId

```

Comment:
Why fetch the DefaultContentTypeID here rather than just discount courses where it's 4?